robottelo/datafactory.py

```diff
@@ -181,6 +181,26 @@ def valid_environments_list():
 
 
 @datacheck
+def valid_hosts_list(env_length=10):
+    """Generates a list of valid host names.
+
+    Note::
+    Host name format stored in db is 'fqdn=' + host_name + '.' + env_name
```

Comment:
I didn't get this format. Generally fqdn=hostname + '.' + domain_name, however here its using env_name.
Could you please share an example of db fqdn ? Just to understand the db format. Thanks